### ISSUE TITLE
Why are Insert operations missing from the MySQL Workload Information?

### ISSUE THREAD

**Issue by HwanheeKim813** [addb-swstarlab/DATATune#25]
Hello,

I noticed that the MySQL Workload Information in your work does not include any workloads involving Insert operations.
Given that your work explores a variety of database workloads, I’m curious to know why Insert operations were not included in the workload configuration.
Additionally, do you plan to address Insert workloads in follow-up research or future experiments?

Thank you in advance for your response!

**Comment by Kwon-sein** [comment]
Hi, 

Thank you for questions about my research.

1. Characteristics of Insert Workloads
Insert operations primarily focus on write-heavy tasks and are significantly influenced by database structure and indexing mechanisms. However, in the evaluated workloads (A, B, E, F), we identified that Insert operations do not serve as a major performance bottleneck. Thus, they were excluded from this specific analysis.

2. Prioritization of Analysis
Our study prioritized analyzing workloads that have a substantial impact on system performance, such as Read, Update, and Read-Modify-Write patterns. Specifically, we focused on workloads with significant Read-Write ratios (A, F) and Update-Scan patterns (E), where the system's behavior and performance are more critical. This led to the lower prioritization of Insert operations in this experiment.

3. Future Research Directions
Insert workloads are planned for inclusion in future studies. Further experiments and analyses will expand on this work to incorporate Insert operations and explore their impact alongside other workload types.

We will make sure to reflect the questions you asked in future research.
